src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -150,13 +150,6 @@ class Browser extends DashboardView {
     this.redirectToFirstClass = this.redirectToFirstClass.bind(this);
 
     this.dataBrowserRef = React.createRef();
-    window.addEventListener('popstate', () => {
```

Comment:
This was added to fix another bug, see https://github.com/parse-community/parse-dashboard/pull/2275. @dblythy It seems to cause navigation issues?

Reply:
The reason this code was added is that the data doesn't reload when going from `http://localhost:4040/apps/abcd/browser/TestObject/qiivFSIcz2/relation` to `http://localhost:4040/apps/abcd/browser/TestObject`. 

It was also added to make the back button when clicking "view relation" work

Reply:
Maybe to replicate the bug, some latency is required for the data to load; if you run dashboard and server on the same machine, you may not experience it. To increase latency you could use a remote server or artificially add latency to your network.

Reply:
This bug is currently blocking the beta release

Reply:
Investigating today

Reply:
changing to:

```
window.addEventListener('popstate', () => {
      this.setState({
        relation: null
      })
    });
```

Seems to fix the issue whilst maintaining back arrow functionality

Reply:
Were you able to replicate the issue?

Reply:
Yes I was on a Parse Server instance running the dashboard in the UK which added some latency, and I observed the bug

Reply:
Nice! Let me see whether I can still observe the issue...

Reply:
Nice! Let me see whether I can still observe the issue with this fix. @damianstasik could you adapt the PR to the suggested changes we can try this out?

Reply:
I've committed the change